MCPForUnity/Server~/tests/integration/test_improved_anchor_matching.py

```
@@ -134,15 +131,33 @@ async def test_apply_edits_with_improved_matching():
     print("Testing improved anchor matching...")
     print("="*60)
 
-    success1 = test_improved_anchor_matching()
+    try:
+        test_improved_anchor_matching()
+        print("test_improved_anchor_matching passed")
+        success1 = True
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)